project_5.html

```diff
@@ -0,0 +1,697 @@
+				
+		// Play the "0001.mp3" file in the "mp3" folder on the microSD card.
+		player.play(1);
+				
+		// Delay the next command by 5 seconds.
+		delay(5000);
+
+		// Pause the playback.
+		player.pause();
+	
+		// Delay the next command by 5 seconds.
+		delay(5000);
+	
+		// Resume the playback.
+		player.start();
+	
+		// Here are some useful debugging functions
+		Serial.println(player.readState()); // Returns some debugging info
+		Serial.println(player.readFileCounts()); // Returns number of files in microSD 
+		Serial.println(player.readCurrentFileNumber()); // Returns current file number being played
+	
+	} else {
+		Serial.println("Connecting to DFPlayer Mini failed!");
+	}
+}
+	
+void loop() {
+	// We didn't need to use the loop here because the program was to only execute once.
+}</code></pre>
+	<h2 class="mb-3 fw-bold" id="pj-heading-arduino-libraries">Installing Arduino Libraries</h2>
+		<p class="mb-3">
+			Here is a quick recap on how you can install Arduino libraries in the
+			<a class="link-primary" href="https://www.arduino.cc/en/software" target="_blank"
+				rel="noopener noreferrer">Arduino IDE 2.0.0+</a>
+		</p>
+		<ol class="mb-4 list-group list-group-numbered">
+			<li class="list-group-item">
+				Download the library as a <strong>.zip</strong>
+			</li>
+			<li class="list-group-item">
+				In the Arduino IDE, go to <strong>Sketch → Include Library → Add .ZIP Library…</strong>
+			</li>
+			<li class="list-group-item">
+				Select the <strong>.zip</strong> of the library and click <strong>Open</strong>.
+			</li>
+		</ol>
+		<div style= "text-align: center;" class="mb-3">
+			<img class="img-fluid my-3 me-5"
+				src=".\assets\images\ops_arduino_ide_zip_library.png" width = "500">
+		</div>
+	
+		<p class="mb-3">
+			You're done! Just make sure to <code class="pj-code language-arduino">#include</code> the library header file so that your
+			code compiles.
+		</p>
+		
+
+			<table class="table pj-download-table mb-5">
+				<tbody>
+					<tr>
+						<td>DFRobot DFPlayerMini</td>
+						<td>
+							<a class="btn btn-outline-success mx-3"
+								href="https://drive.google.com/uc?export=download&id=1tc4KY-tViCPWNbc8Z_jIsfigmyeKCzg4"
+								role="button"><i class="bi bi-download"></i> Download</a>
+						</td>
+					</tr>
+				</tbody>
+			</table>			
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-arduino-nano-pinout" style="display: inline;">Arduino Nano Pinout</h2>
+			<p class="mb-3">
+				Thinking about using a pin but you don’t know what it does? Refer to the Arduino Nano pinout diagram, or scroll further to find the full datasheet.
+				<br><span class="text-blue">~</span> represents pins that are PWM capable.
+			</p>
+		<div class="mb-5 text-center">
+			<img class="img-fluid my-4" src=".\assets\images\pinouts\ops-pinout-arduino-nano.png">
+		</div>
+
+		<h2 class="mb-3 fw-bolder" id="pj-heading-requirements">Requirements</h2>
+		<h3 class="mb-3" style="display: inline;" id="pj-heading-uart-echo">UART Echo</h3>
```

Comment:
Please do not make these headers </h2/> inline. It messes up the margin spacing.

Reply:
This applies to all headers below.